### PR TITLE
Separate checkin and liveonly cortex tests

### PR DESCRIPTION
**Pull request by ritwikbasu**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by ritwikbasu** [comment]
This fixes the issues addressed in https://github.com/Azure/azure-powershell/pull/7378 and https://github.com/Azure/azure-powershell/issues/7260

**Comment by cormacpayne** [comment]
@Nilambari would you mind signing off on this change?

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/CortexTests.cs [hunk 1]
```diff
@@ -45,5 +46,15 @@ public void TestP2SCortexCRUD()
         {
             NetworkResourcesController.NewInstance.RunPsTest(_logger, "Test-P2SCortexCRUD");
         }
```
Comment (maddieclayton) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/CortexTests.cs]: This looks great! Only one small comment - please add the 
```
[Trait(Category.AcceptanceType, Category.CheckIn)]
```
to the TestPS2CortexCRUD test.

Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/CortexTests.cs]: @ritwikbasu Can you help adding this? Thanks!

Comment (ritwikbasu) [reply src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/CortexTests.cs]: Added category type to P2S crud.

**Comment by Nilambari** [comment]
@maddieclayton Can you Please let me know the subscriptions we use for running tests in 
continuous-integration/travis-ci/pr build? Test is failing as there are some feature flags not set. I will work on setting those feature flags on subscriptions so tests wont fail.

**Comment by cormacpayne** [comment]
@Nilambari @ritwikbasu it looks like the `TestP2SCortexCRUD` test failed on the Travis build. Would you mind taking a look?

**Comment by cormacpayne** [comment]
@Nilambari @ritwikbasu Travis and Jenkins use the same recordings when running tests in playback, so there should be no difference in the subscription used between the two builds

**Comment by Nilambari** [comment]
Thanks @cormacpayne and @markcowl for your help in finding the issue.  @maddieclayton Playback issue seems fixed. Can you help merge once build is passed and generate network private module? Thanks!

**Comment by maddieclayton** [comment]
@Nilambari Please fix the cred scan issues: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell/8017/artifact/src/Package/CredentialScannerOutput/CredScanIssues.csv/*view*/

**Comment by Nilambari** [comment]
@maddieclayton Fixed the issues and build is also passed. Can you help merge and generate network private module? Thanks!

**Comment by cormacpayne** [comment]
@maddieclayton would you mind helping to get the private module released?

**Pull request merged**